src/table-view/table-view.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Fragment } from 'react'
+import createReactClass from 'create-react-class'
+import { withStyles, Button } from '@material-ui/core'
+import _ from 'lodash'
+import Select from '../select'
+import Table from '../table'
+
+const TableView = createReactClass({
+  displayName: 'TableView',
+  propTypes: {},
+  getDefaultProps () {
+    return {
+      rows: [],
+      columns: []
+    }
+  },
+  getInitialState () {
+    return {
+      selectedValue: {},
+      selectArray: {},
+      rows: this.props.rows,
+      columns: this.props.columns,
+      selectType: ['genre', 'Rating']
+    }
+  },
+  componentDidMount () {
+    const { selectArray, selectType } = this.state
+    const { rows } = this.props
+    selectType.forEach((type) => {
+      selectArray[type] = _.orderBy(_.uniq(_.map(rows, type.toLowerCase())))
+    })
+    this.setState({ selectArray })
+  },
+  render () {
+    const { classes } = this.props
+    const { selectedValue, selectArray, rows, columns, selectType } = this.state
+    return (
+      <Fragment>
+        <div style={{ display: 'flex' }}>
+          {
+            Object.entries(selectArray).map(([key, value]) => {
+              return (
+                <Select
+                  style={{ flex: 1, maxWidth: '250px', marginRight: '15px' }}
+                  key={key}
+                  items={value}
+                  label={key}
+                  name={key}
+                  multiple
+                  value={selectedValue[key] || []}
+                  onChange={(event) => {
```

Comment:
please refer to the coding styles mentioned on `README`

Reply:
I've taken care of this. I've used the function of oop rather than the arrow function.